### DISCUSSION TITLE
Playlists: keep playlist scores when map gets ranked

### DISCUSSION THREAD

**Discussion by sz-da** [ppy/osu#26845] [Ideas] [answered]
When a map is ranked, leaderboards are wiped. But this also wipes the scores on playlist leaderboards.

The score still counts toward playlist leaderboard
<img src="https://github.com/ppy/osu/assets/76718358/2e106675-37d2-42c0-b134-bd7dc72c5492" width="60%">
but map leaderboard won't show it
<img src="https://github.com/ppy/osu/assets/76718358/4cbc80e1-2e06-4b01-aa0f-a21ecb3c2cd9" width="60%">

edit: leaderboard also appears to be missing the `ⓘ No records yet!` message

**Comment by Rekunan** [comment]
I thought one of the main points of qualified was that scores are temporary, since it acts as a buffer between the map being actually ranked to allow more visibility for any major issues that could lead to a dq or veto. This sort of change feels like undermining that goal.

**Comment by bdach** [comment]
I believe we plan on keeping qualified scores for multiplayer / playlists purposes specifically. This is important for tournaments.

**Reply by Rekunan** [reply]
Ah true forgot about tourney usage.

**Comment by peppy** [comment, marked as answer]
This is being tracked here https://github.com/orgs/ppy/projects/19/views/1?pane=issue&itemId=54442645.
